data/mods/Magiclysm/Spells/druid.json

```
@@ -368,5 +368,33 @@
     "min_duration": 60000,
     "max_duration": 150000,
     "duration_increment": 6000
+  },
+  {
+    "id": "summon_wolf_druid",
+    "type": "SPELL",
+    "name": "Summon Wolf",
+    "description": "Call to the wilds for aid from some wolfs",
+    "valid_targets": [ "ground" ],
+    "min_damage": 1,
+    "max_damage": 3,
+    "damage_increment": 0.07,
+    "min_range": 2,
+    "max_range": 7,
+    "min_aoe": 2,
+    "max_aoe": 2,
+    "range_increment": 0.2,
+    "spell_class": "DRUID",
+    "base_casting_time": 300,
```

```suggestion
    "base_casting_time": 1200,
```
To match with animist summon spells for the relationship between min duration and casting time. Alternatively, it could be made more difficult in other ways.

---

i uped casting time and cost some i hope that feels a little better.